src/main/java/bot/telegram/HttpClientLocal.java

```
@@ -0,0 +1,83 @@
+        } catch (NoSuchAlgorithmException e) {
+            //e.printStackTrace();
+        }
+        try {
+            sslContext.init(null, trustAllCerts, new SecureRandom());
+        } catch (KeyManagementException e) {
+            //e.printStackTrace();
+        }
+        // Create an HttpClient that uses the custom
+        HttpClient httpClient = HttpClient.newBuilder()
+                .version(HttpClient.Version.HTTP_2)
+                .connectTimeout(Duration.ofSeconds(10))
+                .sslContext(sslContext)
+                .build();;
+        try {
+            System.out.println("Call address::::"+address);
+            HttpRequest request = HttpRequest.newBuilder()
+                    .GET()
+                    .uri(URI.create(address))
+                    .build();
+            HttpResponse<String> response = null;
+            try {
+                response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+            } catch (HttpConnectTimeoutException ex){
+                //ex.printStackTrace();
+            } catch (HttpTimeoutException ex){
+                //ex.printStackTrace();
+            } catch (java.net.ConnectException ex){
+                //ex.printStackTrace();
+            } catch (IOException ex){
+                //ex.printStackTrace();
+            }
+
+            System.out.print("scannerAddress::::: "+address);
+
+            if(response != null) {
+                System.out.println(" code::::: "+response.statusCode());
+                return new HttpClientResponse(response.statusCode(),
+                        response.headers().allValues("content-type"),
+                        response.body());
+            }
+            System.out.println(" code::::: 400");
+            return new HttpClientResponse(400, null, null);
+        } catch (InterruptedException ex) {
+            ex.printStackTrace();
+        }
+
+        return null;
+    }
```

The logic in this method is too complex to maintain in a single method. It would be better to split it into separate methods: one dedicated to handling SSL certificates and another for managing the HTTP request. Moreover, avoid silently ignoring exceptions. Instead, log them appropriately, re-throw them if they represent critical failures, or provide a clear comment explaining why ignoring the exception is acceptable in this context.